frontend/src/pages/SignupPage/index.js

```diff
@@ -141,9 +175,15 @@ const SignupPage = () => {
         <Logo />
       </div>
       <div className={styles.main}>{stepInfo[state.step].component}</div>
-      <Button onClick={continueToNextStep}>
-        {stepInfo[state.step].buttonText}
-      </Button>
+      {(() => {
+        if (state.step !== STEP_4) {
```

Comment:
This if exists because the button on the step 4 will be styled differently with the new design

Reply:
So it will come from inside the PrivacyForm component

Reply:
Why the function?

Reply:
Is this what you're looking for?

```
      {state.step !== STEP_4 && (
        <Button onClick={continueToNextStep}>
          {stepInfo[state.step].buttonText}
        </Button>
      )}
```